uber/site_sections/signups.py

```diff
@@ -67,10 +68,13 @@ def volunteer(self, session, id, csrf_token=None, requested_depts='', message='S
         }
 
     @check_shutdown
-    def shifts(self, session):
+    def shifts(self, session, tgtDate='', state=''):
+        joblist = session.jobs_for_signups()
         return {
-            'jobs': session.jobs_for_signups(),
-            'name': session.logged_in_volunteer().full_name
+            'jobs': joblist,
+            'name': session.logged_in_volunteer().full_name,
+            'hours': session.logged_in_volunteer().weighted_hours,
+            'c': c
```

Comment:
Same as above -- you don't need this line since `c` is always available to templates.